mturk/views.py

```diff
@@ -0,0 +1,11 @@
+from django.shortcuts import render
+
+
+def mturk_index(request, *args, **kwargs):
+    from mturk.interface import MTurkProvider
+    from crowdsourcing.models import Project, Task
+    provider = MTurkProvider('https://localhost:8000')
+    project = Project.objects.get(id=1)
```

Comment:
Apologies, all that was for testing only return render belongs there will clean up